src/Application.php

```diff
@@ -104,6 +104,18 @@ protected function initServices() {
             );
         };
 
+        $this['service.calendar'] = function() {
```

Comment:
If we're introducing a calendar / addressbook server instead of using the backends directly, we should use them for everything. I don't think the baikal source should _sometimes_ use the services and _sometimes_ the backend classes. That's not something that needs to be fixed in this PR though.